src/index.html

```diff
@@ -17,11 +17,365 @@
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="link-to-apple-page"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-samsung-page"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-smartphones-page"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-laptopscomputers-page"
+              class="nav__link"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-gadgets-page"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-tablets-page"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-photo-page"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="link-to-video-page"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="content">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo"></div>
+        <div class="card__info">
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__code">Product code: 195434</p>
+          <div class="stars">
+            <div class="stars__container">
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star stars__star--active"></div>
+              <div class="stars__star"></div>
+              <p class="stars__reviews">Reviews: 5</p>
+            </div>
+          </div>
+          <div class="card__price">
+            <span class="card__price-label">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <button
+            class="card__button"
+            data-qa="hover"
```

Comment:
The `data-qa` attribute for the 'Buy' button is incorrectly set to 'hover'. Please update it to the correct value as specified in the task requirements.